conf/application.conf

```diff
@@ -162,6 +162,7 @@ play.filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline'
 gg-urls {
   login-callback.url = "http://localhost:8884/lifetime-isa/company-structure"
   logout-callback.url = "http://localhost:8884/lifetime-isa/signed-out"
+  timeout-callback.url = "http://localhost:8884/lifetime-isa/timed-out"
```

Comment:
Has this new config been overwritten in base or the app-config-env*?

Reply:
Thanks for the reminder! This is a new key-value so will need to add this to each env config.

Reply:
https://github.com/hmrc/app-config-base/pull/6199

Reply:
Just doing base and prod might be easier.

Reply:
https://github.com/hmrc/app-config-production/pull/12549